x-pack/plugins/ml/server/models/data_recognizer/modules/nginx_ecs/kibana/search/ml_http_access_filebeat_ecs.json

```diff
@@ -5,7 +5,7 @@
   ],
   "hits": 0,
   "description": "Filebeat HTTP Access Data (ECS)",
-  "title": "ML HTTP Access Data (ECS)",
+  "title": "ML HTTP Access: Access Data (ECS)",
```

Comment:
Not sure Access having twice here is necessary?

Reply:
On balance, I think I will keep the title of the saved search as-is, with `Access` appearing twice - the first one is from the module description, and the second one describes the search as being for `access` data:

![image 10](https://user-images.githubusercontent.com/7405507/52215238-8ba62f00-288b-11e9-8217-c0210712e8cf.png)